### DISCUSSION TITLE
have you considered excalidrawZ - for its file management functioanalities

### DISCUSSION THREAD

**Discussion by punkyard** [nextcloud/all-in-one#5335] [Ideas] [answered]
<< [Excalidraw](https://github.com/excalidraw/excalidraw) is a very useful web app, but the lack of file management can be troublesome and unsettling. We often need to manually save and maintain multiple different Excalidraw files. 

Therefore, ExcalidrawZ has wrapped it up to automatically save edited files for users and added file grouping functionality. In future versions, iCloud automatic backup will also be added to greatly reduce the insecurity of using a web app. >>
https://github.com/chocoford/ExcalidrawZ

Could it be considered to use ExcalidrawZ instead in excalidraw?

**Comment by szaimen** [comment, marked as answer]
Hi, this is probably better asked at https://github.com/nextcloud/whiteboard
